apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -67,7 +106,7 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
               <select id="user-prompt-example" onchange="setPromptToExample()">
                 <option value=""></option>
                 ${examplePrompts.map(
-                  (question) => html`<option value="${question.question}">${question.id}</option>`,
+                  (question) => html`<option value="${question.id}">${question.id}</option>`,
```

Comment:
As long as we're changing this, it would be nice to use a human-readable summary/title instead of this ID. E.g. "Select the median of 5 random numbers", "Answer the Ultimate Question" (maybe this is not the best example, though it's funny), "Calculate the distance a projective travels).

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483